omrsigcompat/makefile

```diff
@@ -25,6 +25,18 @@ OBJECTS := omrsig
 OBJECTS := $(addsuffix $(OBJEXT),$(OBJECTS))
 EXPORT_FUNCTIONS_FILE := omrsig.exportlist
 
+ifeq (win,$(OMR_HOST_OS))
```

Comment:
We should be linking `omr.res` in all libraries, not just `omrsigcompat`. I would move the link options moved to `rules.mk`